packages/gatsby-plugin-gatsby-cloud/src/__tests__/gatsby-browser.js

```diff
@@ -0,0 +1,161 @@
+import React from "react"
+import "@testing-library/jest-dom"
+import { render, screen } from "@testing-library/react"
+import { wrapPageElement } from "../gatsby-browser"
+import Indicator from "../indicator"
+
+import userEvent from "@testing-library/user-event"
```

Comment:
Mind putting this import up at the top with the rest of the external dependency imports?